service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DirectionAssignee.java

```diff
@@ -17,4 +19,11 @@
     public String getValue() {
         return value;
     }
+
+    public static DirectionAssignee fromString(String value) {
+        return Stream.of(DirectionAssignee.values())
+            .filter(x -> x.getValue().equals(value))
```

Comment:
Does case matter here?

Reply:
so, the keys of our role collections is always the string passed in here. They will always be in the form roleDirections. This will always map to DirectionAssignee

Reply:
method is now removed